### ISSUE TITLE
Your BDD100K instructions are unclear

### ISSUE THREAD

**Issue by iAmJuan550** [SysCV/qdtrack#62]
This is what you are saying:
```

On the official download page, the required data and annotations are

detection set images: Images
detection set annotations: Detection 2020 Labels
tracking set images: MOT 2020 Data
tracking set annotations: MOT 2020 Labels
```
But there is no **Images** or **MOT 2020 Data** on the official website for BDD

**Comment by OceanPang** [comment]
@XiaLiPKU Could you please help to check the update of BDD100K and update the document accordingly?

**Comment by JackieZhai** [comment]
They updated to:

100K Images
Detection 2020 Labels
MOT 2020 Images
MOT 2020 Labels

and the json (coco-format) converters for BDD100k (box_track_20) may not available...

**Comment by iAmJuan550** [comment]
I am closing this issue because I followed @JackieZhai's instructions and successfully downloaded the right BDD100K dataset with labels

**Comment by XiaLiPKU** [comment]
I will update the documents in a new PR today.
Sorry to let you wait

**Comment by OceanPang** [comment]
Fixed in #76
